hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -945,6 +945,9 @@ For the HW_CFG and SECRET* partitions, this can be achieved as follows:
 2. [Read back]({{< relref "#readout-sequence" >}}) and verify the digest location via the DAI.
 3. Perform a full-system reset and verify that the corresponding CSRs exposing the 64bit digest have been populated ({{< regref "HW_CFG_DIGEST_0" >}}, {{< regref "SECRET0_DIGEST_0" >}}, {{< regref "SECRET1_DIGEST_0" >}} or {{< regref "SECRET2_DIGEST_0" >}}).
 
+It should be noted that locking only takes effect after a system reset since the affected partitions first have to re-sense the digest values.
+Hence, SW must ensure that no more data is written to the partition to be locked after triggering the hardware digest calculation.
```

Comment:
If I understand correctly, not doing this will likely brick the device, so maybe just clarify how essential this is?

Reply:
Good point, let me emphasize that.